include/taskolib/SequenceManager.h

```diff
@@ -99,9 +95,47 @@ class SequenceManager
      */
     Sequence load_sequence(std::filesystem::path sequence_path) const;
 
+
+    /**
+     * Delete sequence specified in path to sequence
+     *
+     * \param sequence_path path to sequence.
+     *
+     * \exception throws Error if removal was unsucessful
+     */
+    void remove_sequence(std::filesystem::path sequence_path);
```

Comment:
The function should not allow removing anything else than a sequence (specifically, not "../../usr" or something like that. It should say so in the documentation.